apps/site/pages/dashboard/index.tsx

```diff
@@ -92,96 +119,272 @@ export default function Start(props: Props) {
+							<Card
+								href="/learning-goals"
+								imageElement={<TutorialSvg />}
+								title="Lernziele verwalten"
+							/>
+						</>
+					)}
 				</div>
 			</CenteredSection>
+			<DialogHandler id="studentSettingsDialogDashboard" />
 		</div>
 	);
 }
 
-function Activity({ completedLessons }: { completedLessons: Student["completedLessons"] }) {
+function TagebuchToggle({ onChange }: { onChange: (value: StudentSettings) => void }) {
+	const { data: studentSettings, isLoading, refetch } = trpc.settings.getMySetting.useQuery();
+	const hasLearningDiary = studentSettings?.hasLearningDiary || false;
+	const hasLearningStatistics = studentSettings?.learningStatistics || false;
+
+	return (
+		<>
+			{!isLoading && (
+				<Toggle
+					value={hasLearningDiary && hasLearningStatistics}
+					onChange={() => {
+						dispatchDialog(
+							<StudentSettingsDialog
+								initialSettings={{
+									hasLearningDiary: false,
+									learningStatistics: false,
+									...studentSettings
+								}}
+								onClose={value => {
+									refetch();
+									onChange({
+										hasLearningDiary: value?.hasLearningDiary ?? false,
+										learningStatistics: value?.learningStatistics ?? false
+									});
+									freeDialog("studentSettingsDialogDashboard");
+								}}
+							/>,
+							"studentSettingsDialogDashboard"
+						);
+					}}
+					label="Lerntagebuch"
+				/>
+			)}
+		</>
+	);
+}
+
+function LastLearningDiaryEntry() {
+	const { data: learningDiaryEntries, isLoading } =
+		trpc.learningDiaryEntry.getMeLearningDiaryEntries.useQuery();
+
 	return (
 		<>
-			{completedLessons.length === 0 ? (
+			{learningDiaryEntries && learningDiaryEntries.length == 0 ? (
+				<span className="text-sm text-light">
+					Du hast noch keinen Lerntagebucheintrag erstellt.
+				</span>
+			) : (
+				<>
+					{!isLoading && learningDiaryEntries && (
+						<ul className="flex max-h-80 flex-col gap-2 overflow-auto overflow-x-hidden">
+							{learningDiaryEntries.map((entry, index) => (
+								<Link
+									className="text-sm font-medium"
+									href={`/ltb/entry/${entry.number}/`}
+									key={entry.number}
+								>
+									<li
+										className="hover: flex items-center rounded-lg border border-light-border
+							p-3 transition-transform hover:scale-105 hover:bg-slate-100 hover:shadow-lg"
+									>
+										<div className="flex w-full flex-wrap items-center justify-between gap-2 px-4">
+											<div className="flex flex-col gap-1">
+												<div className="flex items-center gap-1">
+													{index < 2 && (
+														<PencilIcon className="h-5 text-emerald-500" />
+													)}
+													Eintrag {entry.number}. {entry.course.title}
+												</div>
+											</div>
+											<span className="hidden text-sm text-light md:block">
+												{entry.date}
+											</span>
+										</div>
+									</li>
+								</Link>
+							))}
+						</ul>
+					)}
+				</>
+			)}
+		</>
+	);
+}
```

Comment:
sollte zukünftig in LTB library. bitte dran denken

Reply:
kannst du das jetzt noch einmal mit dem feat/ltb/master mergen und das dann in die library schieben?